src/components/track-changes/TrackChangesPanel.tsx

```diff
@@ -37,22 +36,27 @@ export const TrackChangesPanel: React.FC = () => {
     })
   )
 
-  const handleSelect = useCallback((suggestion: TrackedChange) => {
-    setSelectedSuggestion(suggestion, getState)
+  const handleSelect = useCallback((suggestions: TrackedChange[]) => {
+    setSelectedSuggestion(suggestions, getState)
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
   const execCmd = useExecCmd()
 
   const { changeSet } = trackState || {}
 
-  const handleAccept = useCallback((change: TrackedChange) => {
-    setChangeStatus(change, CHANGE_STATUS.accepted, execCmd)
+  const changes = useMemo(
```

Comment:
can you please explain why do we need memo here? thanks

Reply:
removed